### DISCUSSION TITLE
encoder question

### DISCUSSION THREAD

**Discussion by CircuV** [probonopd/MiniDexed#152] [Q&A]
Just got my raspi2 working with minidexed. It's great to have 8 FM7 synths working on a single Raspi.  
Still the encoder is not working that relyable. Is there any way to improve that?  
Sometimes the values do not adjust when tuning the knob, then values change and skip values in between. 
Is there a way to config the encoder? Right now values increase when the encoder is turned
counterclockwise.   I would prefer it the other way round so that values increase when encodeer ist turned clockwise.   Maybe add an entry in the config file to chnage that if possible?

**Comment by craigyjp** [comment]
Doesnt that just mean you have the encoder outer pins the wrong way around for directions. 

I have noticed that encoder can be a little temperamental on speed, sometimes a single click and other times two or four clicks.

**Reply by CircuV** [reply]
Yep thats the way it works sometimes with ym encoder.  
Tnx for the hint, will try changing the pins.

**Comment by luisfcorreia** [comment]
make sure wires are short and well connected

I'm still to suggest additional resistors and capacitors to stabilize

**Comment by probonopd** [comment]
Might be a software thing. Please see https://github.com/probonopd/MiniDexed/issues/89.

If anyone wants to have a go at improving it, that would be great.
